docs/SqlBindingRelatedRepos.md

```diff
@@ -0,0 +1,35 @@
+# Azure SQL Bindings for Azure Functions 
+ 
+## Docs 
+[Azure SQL bindings for Functions | Microsoft Docs ](https://docs.microsoft.com/en-us/azure/azure-functions/functions-bindings-azure-sql?tabs=in-process%2Cextensionv3&pivots=programming-language-csharp)
```

Comment:
Remove en-us

And do you need all the parameters? If you're just linking to the page I'd remove them

Reply:
Ah yes, I can use the aka.ms/sqlbindings link instead